app/modules/backup/management/index.js

```diff
@@ -23,13 +23,15 @@ export default angular.module('backup.management', [
     const mapJobKeyToState = {
       'rollingSnapshot': 'rollingsnapshot',
       'rollingBackup': 'backup',
-      'disasterRecovery': 'disasterrecovery'
+      'disasterRecovery': 'disasterrecovery',
+      undefined: 'index'
```

Comment:
Object keys are automatically converted to string, this is equivalent to `'undefined': 'index'`.

I think it works for your use case, but you should be aware of this.